server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/navbar.ftl

```diff
@@ -61,7 +61,7 @@
             </li>
             <li class="dropdown">
               <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
-                API <span class="caret"></span>
+                Rest <span class="caret"></span>
```

Comment:
REST should be all uppercase, because it's an acronym (REpresentational State Transfer).

Reply:
Roger.  The link being "REST" OK for a name though?

Reply:
Works for me.